PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD002.xml

```diff
@@ -0,0 +1,147 @@
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>            
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1014MD"/>                        
+                        <idno>MD002</idno>            
+
+                    </msIdentifier>
+                    <msContents>
+<msItem xml:id="ms_i1">
+<locus from="1r"/>
+<title xml:lang="gez" type="complete" ref="LIT3186Meeraf">Mǝʿrāf</title>
+<textLang mainLang="gez"></textLang>
+<notatedMusic><desc>Contains musical notation.</desc></notatedMusic>
+    <incipit xml:lang="gez">ምዕራፍ፡ በኩሉ፡ መዋዕል፡ እምዮሐንስ፡ እስከ፡ ዮሐንስ፡ ቅዱስ፡ እግዚአብሔር፡ ቅዱስ፡ ኃያል፡ ቅዱስ፡ ህያው፡
+        ዘኢይመውት፡ ዘተወልደ፡ እማርያም፡ እም፡ ቅድስት፡ ድንግል፡ ተሰዐለነ፡ እግዚኦ፡ ቅዱስ፡ እግዚአብሔር፡ ቅዱስ፡ ኃያል፡ ቅዱስ፡ ህያው፡ ዘኢይመውት፡
+        ዘተጠምቀ፡ በዮርዳኖስ፡ ወተሰቅለ፡ ዲበ፡ እፀ፡ መስቀል፡</incipit>
+    <explicit> በሥርዓተ፡ ጾመ፡ ድጓ፡ መስተብቁዕ፡ በእንተ፡ ሰላም፡ በል፡ ባሕቲቱ፡ ውእቱ፡
+        ዘይውዕል፡ ዐምህላ፡ ቃለ፡ በረከት፡ ዘአቡነ፡ ያሬድ፡ ዘይትነበብ፡ በዕለተ፡ በዕለተ፡ ፫ቱ፡ በ፫ቱ፡ ዕለታት፡
+        ምህላ፡ ምሉዓት፡ ጸሎቱ፡ ዐግድ፡ ፩ከላዋት። </explicit>
+    <note>Church school teachers use the text to teach students in the church school of  
+        <foreign xml:lang="gez">Zemā Bet.</foreign></note> 
+</msItem>
+ </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="paper"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">229</measure>
+                                    <measure unit="leaf" type="blank">3</measure>
+                                    <locus from="228" to="229"/> and <locus target="62v 63r"/>
+                                    <dimensions type="outer">
+                                        <height unit="mm">167</height>
+                                        <width unit="mm">131</width>
+                                        <depth unit="mm">19</depth>
+                                    </dimensions>
+                                </extent>
+                                <foliation resp="AY">Foliation in pencil in the right of the bottom of the recto.</foliation>
+                                <collation>                                    
+                                         <note>I couldn't provide a quire structure here because the material is made of 
```

Comment:
I think, `<collation>` can be deleted here. Otherwise, it is better to use 'could not' instead of 'couldn't', which is spoken English.